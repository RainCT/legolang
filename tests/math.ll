print round(5.4123, 1)
print round(5.9391, 2)

print ""

print pow(10, 5)
print pow(2, 3)
