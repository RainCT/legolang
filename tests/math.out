5.4
5.94

100000.0
8.0
